harness/determined/experimental/pytorch_lightning/__init__.py

```diff
@@ -0,0 +1,4 @@
+from determined.experimental.pytorch_lightning.adapter import (
+    HyperparamsProvider,
```

Comment:
blocking: it doesn't seem very necessary to have this class. Let's just use a dictionary.

Reply:
hmm, dictionary like `Dict[str, any]` ? but it wouldn't match what `context.get_hparams` would it?

Reply:
Why don't just pass in what `get_hparams` returns?